### DISCUSSION TITLE
Adjust each local offset when changing global offset

### DISCUSSION THREAD

**Discussion by AuracleTech** [ppy/osu#20247] [Ideas] [answered]
When I change my global offsets, all my custom local offsets now overshoot/undershoot
Lots of people don't touch global offset due to this reason

Could be nice is if the global offset would affect the local offset

Example : I have +10 ms offset on each single map and set global offset to +5, all offset per map would get replaced by 5 so the total offset did not change ( note :  this would be only for maps with _a defined local offset only_, non custom offset maps would still change )

Windows offset is killing me +- 6ms every time I start the game, I want to be able to gauge it on startup using a calibrating map Id make for myself and set the windows offset on global when starting, without breaking *all* my set local offsets

**Comment by peppy** [comment, marked as answer]
This doesn't make sense. Local offsets should be used to adjust primarily for the beatmap. Therefore they are intentionally treated separately from global.

I'd highly recommend using only the global offset to adjust. Do note that the "windows offset" you mention may change per beatmap or pause, and I'd highly recommend not trying to account for this. A better solution if you are very sensitive to this is to wait for exclusive wasapi support.

**Reply by AuracleTech** [reply]
Ok 😔
Oh and by the way I found an issue while testing things out
I don't think we should be able to pause while playing and change global offset, I was able to skip a big chunk of notes lol kinda funny
